matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/RoomService.kt

```diff
@@ -227,11 +227,14 @@ interface RoomService {
 
     /**
      * TODO Doc.
+     * @param getFlattenParents When true, the list of known parent and grand parent summaries will be resolved.
+     * This can have significant impact on performance, better be used only on manageable list (filtered by displayName, ..).
```

Comment:
Feels weird to have a partial java doc covering only one parameter. I would resolve this TODO and complete the javadoc here (and preferably for the method above too, to avoid confusion later)
```suggestion
     * Gets filtered page of room summaries so that [insert use of filtered here]
     *
     * @param queryParams - tba
     * @param pagedListConfig - tba
     * @param sortOrder - tba
     * @param getFlattenParents When true, the list of known parent and grand parent summaries will be resolved.
```

Reply:
If you use `tba` you should keep the `TODO`